gym_duckietown/wrappers.py

```diff
@@ -37,10 +37,36 @@ def __init__(self, env=None):
         obs_shape = self.observation_space.shape
         self.observation_space = spaces.Box(
             self.observation_space.low[0,0,0],
-            self.observation_space.high[0,0,0],
+            self.observation_space.high[1,1,1],
```

Comment:
Is there a reason that we're specifying the observation space like this?
Can we just hard code 0 and 255?

Reply:
Trying to be more flexible, in case someone ever changed it to [0,1], but yes, we could.

Reply:
No worries then, just wondering. But I'm assuming this is also the line we'd need to change back @nithin127